src/SAML2/XML/ds/KeyInfo.php

```diff
@@ -120,7 +122,7 @@ public function setInfo(array $info)
      */
     public function addInfo($info)
     {
-        assert($info instanceof Chunk || $info instanceof KeyName || $info instanceof X509Data);
+        Assertion::true($info instanceof Chunk || $info instanceof KeyName || $info instanceof X509Data);
```

Comment:
I was pretty sure that it was possible to check if an object was an instance of multiple alternative classes, but then checking into the API, I noticed it wasn't, and I was confused with _webmozart/assert_, which I checked back in the day. Would it make sense to switch to that other library, instead of _beberlei/assert_? The API looks almost the same, and it has the advantage that substitutions in error messages (shall we use them) can have their order specified (and therefore can be easily translated).

Sorry so much that I didn't remember about this before 😞 If it's a lot of work to switch, I'm fine with keeping this one too.